testsuite/shared/src/main/java/org/jboss/as/test/config/ContainerConfig.java

```diff
@@ -0,0 +1,270 @@
+package org.jboss.as.test.config;
+
+import org.testcontainers.shaded.com.google.common.base.Strings;
```

Comment:
This usage feels fragile to me. It seems only `isNullOrEmpty()` is used and we could easily implement that where with:
```java
private static boolean isNullOrEmpty(final CharSequence cs) {
    return cs == null || cs.length() == 0;
}
```